docs/user_profile_core_plus_extended.yml

```diff
@@ -0,0 +1,558 @@
+      created: '2018-01-01T00:00:00Z'
+      publisher_authority: access_provider
+    values:
+      5a5munnfxYjqkaN0su1Kl7USxbqkILQN: 7776000
+
+## Starting here are the EXTENDED profile fields. ALL extended profile fields are OPTIONAL.
+
+# `fun_title.value` is a title the user gives to themselves, not their official company title. Anyone can have a title.
+fun_title:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  value: Dum own title
+
+# `description.value` is a free-form text field for user's own description.
+description:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  value: I am a funny profile!
+
+# `location_preference.value` is the geographical location that the user wants to appear as.
+location_preference:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  value: Bahamas
+
+# `office_location.value` is the Mozilla Office the user wants to show they're usually at.
+office_location:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  value: San Francisco
+
+# `timezone.value` is the timezone the user wants to appear as.
+timezone:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  value: UTC
+
+# `preferred_languages.values[]` is a list of languages the user prefers communicating with.
+preferred_languages:
+  signature:
+    publisher:
+      alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+    additional:
+    - alg: 'RS256'
+      typ: 'JWT'
+      value: ''
+  metadata:
+    classification: PUBLIC
+    last_modified: '2018-01-01T00:00:00Z'
+    created: '2018-01-01T00:00:00Z'
+    publisher_authority: mozilliansorg
+  values:
+    - French
+      English (US)
```

Comment:
this should be a list:
```yaml
- French
- English (US)
- German
```

Reply:
No it's not :confused: (or am I missing something here?).

Quoting [yaml.org](http://yaml.org/spec/1.2/spec.html#id2797293):
> A block sequence is simply a series of nodes, each denoted by a leading “-” indicator.

It also renders wrong in the according [JSON](https://github.com/mozilla-iam/cis/pull/112/files#diff-63a3d9c31029bcc54e825c6c3fcbf858R543) which is generated from this YAML, isn't it?

Reply:
the yaml is generated from the json, though i might have messed something up ill check that

Reply:
so yeah my bad, this is a list not a list of dict, fixing :)